www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -1,6 +1,6 @@
 <model>
   <mount>//OPNsense/Nginx</mount>
-  <version>1.32</version>
+  <version>1.32.1</version>
```

Comment:
I'd like to cram this into 1.32 as it hasn't been released yet.

```suggestion
  <version>1.32</version>
```

Reply:
sure. maybe i should update `pkg-descr` then?

Reply:
indeed 👍